src/Mono.Android/Android.Runtime/JNIEnvInit.cs

```diff
@@ -82,6 +82,10 @@ static Type TypeGetType (string typeName) =>
 		internal static void InitializeJniRuntime (JniRuntime runtime)
 		{
 			androidRuntime = runtime;
+			gref_gc_threshold = RuntimeNativeMethods._monodroid_max_gref_get ();
+			if (gref_gc_threshold != int.MaxValue) {
+				gref_gc_threshold = checked((gref_gc_threshold * 9) / 10);
+			}
```

Comment:
Is this taking 90% of this number for all runtimes? Do we need this change?

Reply:
Other runtimes set `gref_gc_threshold` through different code path but it's still the same value (90% of the max GREF count).